tests/integration/conftest.py

```diff
@@ -674,3 +719,109 @@ def kernel(shell):
 
         # Reboot after clean-up
         shell("tmt-reboot -t 600")
+
+
+@pytest.fixture()
+def yum_conf_exclude(shell, request):
+    """
+    Fixture.
+    Define `exclude=kernel kernel-core` in /etc/yum.conf.
+    Using pytest.mark.parametrize pass the packages to exclude in a single scenario as a list.
+    Call the fixture directly in the test function.
+    Example:
+        # one testcase
+        @pytest.mark.parametrize("exclude", [["this", "that", "also_this"]])
+
+        # more testcases
+        @pytest.mark.parametrize("exclude", [["this", "that"], ["then_this"]])
+
+        def test_function(yum_conf_exclude, exclude):
+            yum_conf_exclude(exclude)
+    """
+    yum_config = "/etc/yum.conf"
+    backup_dir = "/var/tmp/config-backup"
+    config = configparser.ConfigParser()
+    config.read(yum_config)
+
+    shell(f"mkdir {backup_dir}")
+
+    assert shell(f"cp {yum_config} {backup_dir}").returncode == 0
+    # If there is already an `exclude` section, append to the existing value
+    def _exclude_packages(exclude):
+        for pkg in exclude:
+            if config.has_option("main", "exclude"):
+                pre_existing_value = config.get("main", "exclude")
+                config.set("main", "exclude", f"{pre_existing_value} {pkg}")
+            else:
+                config.set("main", "exclude", pkg)
+
+        with open(yum_config, "w") as configfile:
+            config.write(configfile, space_around_delimiters=False)
+
+        assert config.has_option("main", "exclude")
+        assert all(pkg in config.get("main", "exclude") for pkg in exclude)
+
+    yield _exclude_packages
+
+    # Clean up
+    assert shell(f"mv {backup_dir}/yum.conf {yum_config}").returncode == 0
+    assert shell(f"rm -r {backup_dir}").returncode == 0
+
+
+def _add_client_tools_repo_oracle(shell):
+    """
+    Helper function.
+    Runs only on Oracle Linux system
+    Create an ubi repo for its respective major version to install subscription-manager from.
+    """
+    if "oracle" in SYSTEM_RELEASE_ENV:
+        with open("/etc/system-release", "r") as sysrelease_file:
+            content = sysrelease_file.read()
+            sysver = get_system_version_number(content)
```

Comment:
this could also be replaced by the https://github.com/oamg/convert2rhel/pull/1192/files#diff-c87c5173e88bb0f60b00107f4586304f4869a976a4f998425efe3481c0db9f56R299